pkgs/tools/security/exploitdb/default.nix

```diff
@@ -11,11 +11,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-gUjFFxzkHHhNMDAgFmmIAuEACSCn1YXuauvjGAkrK6k=";
   };
 
+  buildInputs = [ makeWrapper ];
+
   installPhase = ''
     runHook preInstall
     mkdir -p $out/bin
-    cp --recursive ./* $out/bin
-    cp ./.searchsploit_rc $out/bin
+    makeWrapper ${src}/searchsploit $out/bin/searchsploit
```

Comment:
Please copy the script to bin otherwise a reference to src is always required and can't be garbage collected.

Reply:
The entire src is required, as the script depends on other files from $src being in the same directory as itself. Should I just copy the entire $src to $out/share and wrap that instead?

Reply:
I updated the package. Now the one script goes in $out/bin and all the other needed files go in $out/share/exploitdb.